docs/gen-docs/kyma_provision_gke.md

```diff
@@ -0,0 +1,41 @@
+## kyma provision gke
```

Comment:
We probably have to manually delete the old docs file `kyma_provision_gcp.md`

Reply:
done